src/Sentry/Internal/Hub.cs

```diff
@@ -230,6 +293,18 @@ public void CaptureTransaction(Transaction transaction)
             }
         }
 
+        public void CaptureSessionSnapshot(SessionSnapshot sessionSnapshot)
```

Comment:
Can we get away without this being public?

Reply:
Not to my knowledge